### PR TITLE
Bug 1179194 - [Stingray][Notification] Initial commit

### PR DESCRIPTION
**Pull request by rexboy7**
Things done in this bug:
- Skeleton of notification app(most of the codes, with bower components)
- IAC skeleton in smart-system app and notification app

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c4fd04855aed10a15866d0378ed913aa2d94e6af

File: tv_apps/notification/js/notification.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+window.addEventListener('iac-notification-message', function(evt) {
```
Comment (john-hu) [inline tv_apps/notification/js/notification.js]: You forget to include `iac_handler.js`. So, you won't have such kind of event. If you don't want it, you can use mozSetMessageHandler('message') to handle it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a00ff0ff8996843d25eb66a6db0d44df4301ea3c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=723a0c41a30a7c445da1428da774355ea457250c

**Pull request merged**